### PR TITLE
Use pause:3.2 image which has arm and amd versions

### PR DESCRIPTION
**Pull request by steezeburger**
The latest `pause` image is quite old and does not support arm.
https://github.com/linkerd/linkerd2/issues/7558

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,5 @@
 target/
+
+
+# jetbrains
+.idea
```
Comment (SuperFluffy) [inline .gitignore:5]: Quick comment on the draft: can you put this bit into a separate PR?

Comment (steezeburger) [reply .gitignore:5]: I understand wanting to keep PRs succinct and indexable, but I think this is fine in this instance. It's not related to our own code at all and feels excessive to spend extra time breaking it out into another PR.

**Review by SuperFluffy — APPROVED** [review]
That's a good find and apologies I didn't test this on macOS.
